tee/prototype/epochproc.go

```diff
@@ -94,6 +94,13 @@ func (e *Enclave) txRoutine(
 			if err := noInconsistentExits(&stagedTxs, exiters); err != nil {
 				log.Errorf("txRoutine: checking exiters: %v", err)
 			}
+
+			log.Infof("txRoutine: removing %d exiters", len(exiters))
+			for _, exiter := range exiters {
+				log.Infof("txRoutine: removed %s", exiter.Hash().String)
+				delete(txEpoch.balances, exiter)
+			}
+
```

Comment:
I think this should be a method `Epoch.applyExits` so that an `Enclave` method doesn't work on implementation details of an `Epoch`.